File: testtasksmart.client/src/DTO/Project.ts
import { PartnerDTO } from "./PartnerDTO";

export interface Project {
  id: number;
  formattedId: string;
  type: string;
  startDate: Date;
  endDate: Date;
  projectManager: PartnerDTO;
  comment: string;
  status: boolean;
}

File: TestTaskSmart.Server/DataAccess/Models/Project.cs
namespace TestTaskSmart.Server.DataAccess.Models
{
    public class Project
    {
        public int Id { get; set; }

        public ProjectTypes Type { get; set; }

        public DateTime StartDate { get; set; }

        public DateTime EndDate { get; set; }

        public int ProjectManagerId { get; set; }
        public Employee ProjectManager { get; set; }

        public ICollection<Employee> Employees { get; set; }

        public string Comment { get; set; }

        public bool Status { get; set; }
    }

    public enum ProjectTypes
    {
        SoftwareDevelopment = 0, 
        ResearchAndDevelopment = 1,
        Maintenance = 2,
    }
}
